pkg/daemon/daemon.go

```diff
@@ -346,6 +348,11 @@ func (dn *Daemon) operatorConfigChangeHandler(old, new interface{}) {
 		glog.Infof("Set log verbose level to: %d", level)
 		flag.Set("v", level.String())
 	}
+	newDisableDrain := *newCfg.Spec.DisableDrain
```

Comment:
This will fail as the initial value for disableDrain (if omitted) is nil.
We could either use a boolean value(not pointer) for DisableDrain (since we don't have to check the existence) or check whether it's null.

Reply:
Correct, I used the pointer as an anology with other boolean values in `SriovOperatorConfigSpec`, fixed